test/minikube/minikube_domain_resync_test.go

```diff
@@ -311,6 +377,43 @@ func TestDomainResync(t *testing.T) {
 	}, 300*time.Second)
 }
 
+func TestLoadBalancerConfigurationResync(t *testing.T) {
+	testlib.AwaitTillerUp(t)
+	defer testlib.VerifyTeardown(t)
+	defer testlib.Teardown(testlib.TEARDOWN_ADMIN)
+
+	options := &helm.Options{
+		SetValues: map[string]string{
+			"admin.lbConfig.prefilter":              "not(label(region tiebreaker))",
+			"admin.lbConfig.default":                "random(first(label(node node1) any))",
+			"admin.lbConfig.policies.zone1":         "round_robin(first(label(zone zone1) any))",
+			"admin.lbConfig.policies.nearest":       "random(first(label(pod ${pod:-}) label(node ${node:-}) label(zone ${zone:-}) any))",
+			"admin.lbConfig.fullSync":               "true",
+			"database.sm.resources.requests.cpu":    "0.25",
```

Comment:
you can probably give it `MINIMAL_VIABLE_ENGINE_CPU` since you are only starting 2 engines.